opinions.yml

```diff
@@ -2,6 +2,7 @@
 properties:
   cf_mysql:
     mysql:
+      startup_timeout: 300
```

Comment:
We "took it out" because it changed name and fissile complained that it couldn't find it.
I don't think the change to _remove_ was deliberate.